### PR TITLE
Convert constants to constexpr

### PR DESCRIPTION
**Pull request by ldicker83**
Does a bit of refactoring in naming schemes and definition of constants prefering constexpr where applicable.

This is a bit of a poorly named branch because it's actually a refactor more than anything else.

Also, this is going to seem like a HUGE changeset, but it really isn't. Lots of lines changed to reflect improved naming conventions.

### CODE REVIEW

**Comment by Brett208** [comment]
I appreciate pulling the underscores from the naming convention here. :)

**Comment by ldicker83** [comment]
> I appreciate pulling the underscores from the naming convention here. :)

Woot!

Yeah, it's been bugging me that these have sat around like this for a long time after we decided that we weren't going to use ALL_CAPS_NAMES like that anymore.

Once the checks finish I'm going to merge into master and call it a night. :D

File: OPHD/Constants/Numbers.h [hunk 1]
```diff
@@ -8,18 +8,18 @@
  */
 namespace constants
 {
-	const int FADE_SPEED = 300;
+	inline constexpr int FADE_SPEED{ 300 };
```
Comment (DanRStevens) [inline OPHD/Constants/Numbers.h]: Btw, `inline` is implied by `constexpr`. You only need to specify `constexpr`.

**Review by DanRStevens — COMMENTED** [review]
Yes!

I am so happy to see the CamelCase change.

File: OPHD/Constants/Numbers.h (hunk 1 above)
Comment (ldicker83) [reply OPHD/Constants/Numbers.h]: That's not what the article I was reading said, but reading the actual standard does confirm this. Wherps. Oh well, I'll worry about it another time. :D

**Pull request merged**